src/tools/roc_recv/cmdline.ggo

```diff
@@ -43,7 +43,7 @@ section "Options"
         int optional
 
     option "frame-size" - "Internal frame size, number of samples"
-        int optional
+        string optional
```

Comment:
We should also provide typestr:

```
typestr="TIME"
```

The `TIME` format is described below in the help text:

```
TIME is an integer number with a suffix, e.g.:
  123ns; 123us; 123ms; 123s; 123m; 123h;
```

BTW it's not present in roc-conv help yet, we should add it too.

Reply:
done

Reply:
done.